### PR TITLE
[Test] Add ml singleshot example

### PR DESCRIPTION
**Pull request by zhoonit**
## Commits to be reviewed in this PR

<details><summary>[Test] Add ml singleshot example</summary><br />

This patch adds integrated test to show and test ml singleshot.

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: Jihoon Lee <jhoon.it.lee@samsung.com>

</details>

### CODE REVIEW

File: test/nnstreamer/test_nnstreamer_single.cpp [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+#include <nnstreamer-single.h>
+#include <nnstreamer.h>
+
+#include <nntrainer_test_util.h>
+
+static std::string mnist_model_path =
+  getResPath("mnist.ini", {"test", "test_models", "models"});
+
+TEST(mlInference, singleshot_p) {
```
Comment (myungjoo) [inline test/nnstreamer/test_nnstreamer_single.cpp:23]: add negative cases. (you need to keep the number of negative > the number of positive)

Comment (zhoonit) [reply test/nnstreamer/test_nnstreamer_single.cpp:23]: updated :)

**Review by jijoongmoon — APPROVED** [review]
LGTM

File: test/nnstreamer/test_nnstreamer_single.cpp [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+  ml_single_h single;
+  ml_tensors_info_h in_info, out_info;
+  ml_tensors_info_h in_data, out_data;
+  int status = 0;
+
+  // 1. input preparation, in this example, we are not filling actual data
+  status = ml_tensors_info_create(&in_info);
+  EXPECT_EQ(status, 0);
```
Comment (kparichay) [inline test/nnstreamer/test_nnstreamer_single.cpp:31]: `/s/EXPECT_EQ(status, 0);/EXPECT_EQ(status, ML_ERROR_NONE);`

**Review by kparichay — APPROVED** [review]
LGTM

**Review by myungjoo — APPROVED** [review]

**Pull request merged**